Scripts/NeedDisabledSVE.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import re
+import sys
+import tempfile
+import platform
```

Comment:
Not all of these seem to be used, could you double-check which are actually needed?